docs/configuration.rst

```diff
@@ -104,6 +104,25 @@ ACCOUNT_USERNAME_REQUIRED (=True)
   `ACCOUNT_AUTHENTICATION_METHOD` is set to `email`. Set to `False`
   when you do not wish to prompt the user to enter a username.
 
+ACCOUNT_LOGIN_CALLBACK_PROXY (="")
+  Redirect the login callback to this remote server or endpoint, instead of the
+  default endpoint on the local server. The remote server should be a second
+  instance of the local server, at a dedicated URL. By enabling
```

Comment:
probably better to relax to the actual constraint and then provide this as a suggestion, then.